maintainers/working_groups/learning_journey/tutorial-outlines/first-steps-trivial-builders.md

```diff
@@ -0,0 +1,37 @@
+# First Steps - Trivial Builders
+
+## Assumptions
+
+* The reader knows what a shell environment is.
+* The reader has some familiarity with `nix-shell`
+
+## Activity
+
+* Start with a `shell.nix` which brings in some packages and dependencies for a `Process X`
```

Comment:
more detail on process x - how can we make it more concrete? A contributor should be able to start with as few questions as possible.